### DISCUSSION TITLE
Workaround for Nerd Fonts 3.0.0 extra space?

### DISCUSSION THREAD

**Discussion by GethDeeo** [JanDeDobbeleer/oh-my-posh#3780] [Troubleshoot]
After upgrading to Nerd Fonts 3.0.0 and updating my theme with the new code points, I've noticed an extra space without background at the **end** of segments containing updated code points (the problematic icon is not at the end).
(Well, except `path` segment, where the same bug appears after the folder icon, just before the _actual path_.)
I've tried both json (example: `\udb80\udd8d`) and yaml (example: `\U000f018d`).

Oh My Posh 15.4.0
Microsoft Terminal 1.16.10262.0

I'm fairly certain it's a Terminal bug. Are there any workarounds, or should I just downgrade the font?

**Comment by JanDeDobbeleer** [comment]
@GethDeeo this is indeed a Windows Terminal bug. There's a possible workaround in the [FAQ](https://ohmyposh.dev/docs/faq#windows-terminal-unexpected-space-between-segmentstext).

**Reply by GethDeeo** [reply]
Thanks for the suggestions, my theme is already using them extensively. Unfortunately, neither of the two hinted characters are helping with these newly popped "voids", I've tried a fair number of combinations :(
Downgraded the font in the meantime.
